pkg/api/types.go

```diff
@@ -703,6 +703,11 @@ type ISCSIVolumeSource struct {
 	// The secret is used if either DiscoveryCHAPAuth or SessionCHAPAuth is true
 	// +optional
 	SecretRef *LocalObjectReference
+	// Optional: Custom initiator name per volume.
+	// If InitiatorName is specified with ISCSIInterface simulateneouly, new iSCSI interface
+	// <target portal>:<volume name> will be created for the connection.
+	// +optional
+	InitiatorName string
```

Comment:
Does this change reduce size of ISCSIVolumeSource struct?
I'd like to know the reason for the future reference.